lib/libyui_client.rb

```diff
@@ -1,49 +1,39 @@
 # frozen_string_literal: true
 
+require 'json'
+require 'net/http'
+require 'uri'
+require 'timeout'
+
 require 'libyui_client/version'
-require 'libyui_client/widget_controller'
-require 'libyui_client/process_helper'
-require 'libyui_client/widget'
-require 'libyui_client/rest_api_client'
+require 'libyui_client/local_process'
+require 'libyui_client/widgets'
```

Comment:
Could this belong to `app` instead? I see that you created `widgets.rb` with all the `require`s,maybe you can help me to understand why that file is needed?